portfolio/src/main/webapp/comments.js

```diff
@@ -17,6 +17,7 @@
  * submits a GET request to /data, receives the response
  * and displays it on the page
  */
+
```

Comment:
nit: no need to have a blank line between function comment and function definition.